packages/common-integration-tests/src/common-int-test-utils.ts

```diff
@@ -21,6 +21,18 @@ import {
 import {v4} from 'uuid';
 import {ITopicClient} from '@gomomento/sdk-core/dist/src/clients/ITopicClient';
 
+const isInsideGithubCI = process.env.CI === 'true';
```

Comment:
Does GH set this?

Reply:
yea they set it here https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables